student_guide.md

```diff
@@ -136,22 +119,20 @@ the links in them!***
 > This guide assumes you already have Eclipse with EGit installed.
 > If you need to install it, follow the [installation guide](/installguides).
 
+> If you are on the lab machines, skip steps 2, 3, and 4 here.
+
 1. Launch Eclipse by double clicking the icon.
 2. In the dialog box that appears, leave the default value and check the box
    in the lower left corner to have this set as the default. Then click OK.
 3. In the upper right corner of the new Eclipse window, click ***Workbench***.
 4. On the toolbar go to ***Window > Perspective > Open Perspective > Java***.
-5. Add the Terminal to the view by clicking !["terminal"](/images/terminal.png)
-   in the toolbar. Then click OK in the dialog box that appears.
-
-    > The terminal that appears will appear in the same tab as the java
-    > debugger that you will rely on throughout the semester, so be sure to
-    > close the terminal when you are not using it (or switch tabs)!
-
-6. Now, add the EGit plug-in to the workbench by clicking ***Window > Show
+5. Now, add the EGit plug-in to the workbench by clicking ***Window > Show
    View > Other... > Git > Git Repositories***.
+6. Add the Git menu to your toolbar by clicking ***Window > Customize Perspective...
+   > Command Groups Availability (tab)***. Check **Git** and **Git Navigation Actions**.
+   Click ok to finish
 
-At this point, Eclipse should look like [this](/images/env_preview.png).
+At this point, Eclipse should look like [this]().
```

Comment:
@ericthewry This is the line causing the build to fail. There should be a new screenshot in the `unprocessed_screenshots` folder to replace this missing link.